frontend/src/layouts/EditPublishLayout.tsx

```diff
@@ -25,6 +25,34 @@ const EditPublishLayout: React.FC<EditPublishLayoutProps> = ({
     if (e.key === 'Enter') writingPadRef.current?.focus();
   };
 
+
+
+// Register the custom video handler with Quill toolbar
+Quill.register('modules/customToolbar', function (quill: any) {
+  quill.getModule('toolbar').addHandler('video', videoHandler.bind(quill));
+});
+
+ 
+
+
+
+
+  
+  const modules = {
+    toolbar: {
+      container: [
+        [{ 'header': '1' }, { 'header': '2' }, { 'font': [] }],
+        [{ 'list': 'ordered'}, { 'list': 'bullet' }, { 'list': 'check' }], 
+        ['bold', 'italic', 'underline', 'strike'],
+        [{ 'align': [] }],
+        [{ 'color': [] }, { 'background': [] }],
+        ['code-block'],
+        ['link', 'image', 'video'],
+        ['clean']
+      ],
+    },
+  };
```

Comment:
The modules should also be defined once, and reused in Publish & Edit, shouldn'it ?

Reply:
Yes thought the same thing, but was not able to find any file containing constants
Can you just tell me where all the constants for the front end are stored?